client/src/App.js

```diff
@@ -48,9 +48,10 @@ class App extends Component {
         admin: true
       });
     });
-    socket.on("startGame", () => {
+    socket.on("startGame", startingPlayer => {
```

Comment:
Could you move it to be emitted as part of the `gameInfo` message to each player?

`gameInfo` is intended to be game implementation specific, so the client-side can render a UI per "game". Some games won't have a starting player.

Reply:
I've taken it out of here and moved it specific to the `gameInfo` emit